build.gradle

```diff
@@ -57,7 +59,7 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     testCompile group: 'org.mockito', name: 'mockito-core', version: '2.24.0'
     zip "sonarqube:sonarqube:${sonarqubeVersion}@zip"
-    compileOnly group: 'org.sonarsource.sonarqube', name: 'sonar-plugin-api', version: "7.7"
+    compileOnly group: 'org.sonarsource.sonarqube', name: 'sonar-plugin-api', version: "${sonarqubeVersion}"
```

Comment:
I suspect this line can actually be removed. It was enforced by the `sonarPackaging` plugin stating that `sonar-plugin-api` is required, but the full SonarQube API/internals are already included through the `sonarLibraries` reference above.

Reply:
It worked fine without the dependency as well. PR updated